Add blogpost announcing Community Awards (#5482)
open-telemetry/opentelemetry.io: content/en/blog/2024/community-awards.md

---
title: 2024 OpenTelemetry Community Awards
linkTitle: 2024 OpenTelemetry Community Awards

> date: 2024-10-25

Reviewer: ```suggestion
date: 2024-10-29
```

author: OpenTelemetry Governance Committee
---

> OpenTelemetry is a community-driven project. A community of awesome humans who
> are actively revolutionizing the field of observability with their
> contributions. Whether it's through code, documentation, project management,
> outreach, adoption, or simply helping others answer technical questions on our
> [CNCF Slack](https://slack.cncf.io/). We want to **recognize these contributions
> and the people behind them**, because we're all human, and we all like that warm
> fuzzy feeling of appreciation.

Reviewer: ```suggestion
OpenTelemetry is a community-driven project, fueled by a group of awesome humans who
are actively revolutionizing the field of observability with their
contributions. Whether it's through code, documentation, project management,
outreach, adoption, or simply helping others answer technical questions on our
[CNCF Slack](https://slack.cncf.io/), we want to recognize these contributions
and the people behind them. Because we're all human, and we all like that warm
fuzzy feeling of appreciation.
```

We are thrilled to announce the first-ever **OpenTelemetry Community Awards**!
This is your chance to nominate individuals who have made a notable impact to
OpenTelemetry over the past year. _Everyone_ can nominate _anyone_ in the

> community, be it a contributor, end-user, or community member.

Reviewer: ```suggestion
community, be they a contributor, end-user, or community member.
```

> Please submit nominations **via the following
> [Google Form](https://forms.gle/ioqFNmDhKNYYAtRs7).**

Reviewer: ```suggestion
Please submit nominations using this
[Google Form](https://forms.gle/ioqFNmDhKNYYAtRs7).
```

> Although we require sign-in to ensure one nomination only per person,
> nominations are completely anonymous. You can nominate up to five people, and
> nominations will be open **until midnight UTC on November 8th, 2024**. Winners
> will be announced at KubeCon + CloudNativeCon North America 2024.

Reviewer: I think this is the way it works, but correct me if I'm wrong.

```suggestion
Although we require sign-in to ensure only one form submission per person,
nominations are completely anonymous. You can nominate up to five people per submission, and
nominations will be open **until midnight UTC on November 8th, 2024**. Winners
will be announced at KubeCon + CloudNativeCon North America 2024.
```

---
title: 2024 OpenTelemetry Community Awards

> linkTitle: 2024 OpenTelemetry Community Awards

Reviewer 2: ```suggestion
linkTitle: OpenTelemetry Community Awards
```

date: 2024-10-29
author: OpenTelemetry Governance Committee
---

OpenTelemetry is a community-driven project, fueled by a group of awesome humans
who are actively revolutionizing the field of observability with their
contributions. Whether it's through code, documentation, project management,
outreach, adoption, or simply helping others answer technical questions on our
[CNCF Slack](https://slack.cncf.io/), we want to recognize these contributions

> and the people behind them. Because we're all human, and we all like that warm

Reviewer 2: "Because ..." isn't a complete sentence. How about:

```suggestion
and the people behind them -- because we're all human, and we all like that warm
```

fuzzy feeling of appreciation.

We are thrilled to announce the first-ever **OpenTelemetry Community Awards**!
This is your chance to nominate individuals who have made a notable impact to
OpenTelemetry over the past year. _Everyone_ can nominate _anyone_ in the
community, be they a contributor, end-user, or community member.

> Please submit nominations using this
> [Google Form](https://forms.gle/ioqFNmDhKNYYAtRs7).

Reviewer 2: ```suggestion
Submit nominations using
[this Google Form](https://forms.gle/ioqFNmDhKNYYAtRs7).
```
